container/src/LuigiCompoundContainer.svelte

```diff
@@ -103,6 +103,9 @@
     thisComponent.updateContext = (contextObj: any, internal?: any) => {
       mainComponent._luigi_mfe_webcomponent.context = contextObj;
     };
+    thisComponent.getContext = (): any => {
+      return mainComponent?._luigi_mfe_webcomponent?.context || {};
+    };
```

Comment:
I would avoid introducing a new feature in this Test-only PR. This **getContext** function would then also have to be documented and shipped as a new feature. You can create a new Issue and ask if it makes sense to add it at this point. Thereafter you may have a separate PR for that. Since we havent added it till now, there probably were no requests so far. 

Another potential issue is clashing with this Light DOM Container PR [change here](https://github.com/SAP/luigi/pull/3697/files#diff-9ef147170f1d5c78725fc1fe4994ed87b810dde97fbc0f2b91ef886fe136db4eR114)- where mainComponent will depend on the **noShadow** option

This might work but there might be another way to grab the context. 
I believe since updateContext was made available for CompoundContainer, there should have already been a way to get the context. I am thinking some way through container.getAttribute('context') maybe or another form like **get/set context(ctx)** WC functions  . If there isnt a very simple way then you can skip adding this test for compound for now.